### PR TITLE
Add QR code when doing a network preview to facilitate accessing the testing url

### PR DESCRIPTION
**Pull request by AlexandreSi**
image

### CODE REVIEW

**Comment by 4ian** [comment]
It's beautiful

File: newIDE/app/src/Export/LocalExporters/LocalPreviewLauncher/LocalNetworkPreviewDialog.js [hunk 1]
```diff
@@ -4,94 +4,109 @@ import { Trans } from '@lingui/macro';
...
-              <Text>
-                <Trans>
-                  Please note that your device should be connected on the same
-                  network as this computer.
-                </Trans>
-              </Text>
+          <ShareLink url={url} />
+          <Line>
+            <Text>
+              <Trans>Or flash this QR code:</Trans>
```
Comment (ClementPasteau) [inline newIDE/app/src/Export/LocalExporters/LocalPreviewLauncher/LocalNetworkPreviewDialog.js:90]: maybe but this line inside the check on `urlWithProtocol`

File: newIDE/app/src/GameDashboard/ShareDialog.js [hunk 2]
```diff
@@ -16,7 +15,6 @@ import ShareButton from '../UI/ShareDialog/ShareButton';
 type Props = {| game: Game, onClose: () => void |};
 
 const ShareDialog = ({ game, onClose }: Props) => {
-  const [showCopiedInfoBar, setShowCopiedInfoBar] = React.useState(false);
```
Comment (ClementPasteau) [inline newIDE/app/src/GameDashboard/ShareDialog.js:19]: oops

File: newIDE/app/src/UI/QrCode.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    () => {
+      const containerElement = containerRef.current;
+
+      if (!containerElement) return;
+
+      QrCreator.render(
+        {
+          text: url,
+          radius: 0,
+          ecLevel: 'M', // L, M, Q, H
```
Comment (ClementPasteau) [inline newIDE/app/src/UI/QrCode.js:26]: reasoning behind this value?

File: newIDE/app/src/UI/QrCode.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  const gdevelopTheme = React.useContext(GDevelopThemeContext);
+
+  const containerRef = React.useRef<?HTMLDivElement>(null);
+  React.useEffect(
+    () => {
+      const containerElement = containerRef.current;
+
+      if (!containerElement) return;
+
+      QrCreator.render(
```
Comment (ClementPasteau) [inline newIDE/app/src/UI/QrCode.js:22]: that's a weird way of rendering itself... too bad we can't just call it where we want it to be displayed

**Review by ClementPasteau — APPROVED** [review]

File: newIDE/app/src/UI/QrCode.js (hunk 3 above)
Comment (AlexandreSi) [reply newIDE/app/src/UI/QrCode.js:26]: It changes the size of the qr code. I'll add a comment.

File: newIDE/app/src/UI/QrCode.js (hunk 4 above)
Comment (AlexandreSi) [reply newIDE/app/src/UI/QrCode.js:22]: There was some react libraries but they were a bit too heavy/overkill

**Pull request merged**